shared/lib/state_manager/state_manager.js

```diff
@@ -99,9 +117,16 @@ export default class StateManager {
   getInitialData(){
     let state_manager = this;
     // we'll load past user answers and get CC results here.
+    state_manager.checkLocalStorage();
     return state_manager.updateDefaults();
```

Comment:
This should be either or. If there's data in localStorage, just load the localStorage values, no need to call api again (return `Promise.resolve()` if need be).

Otherwise, just make the api call and answers should be saved.

We may also want to add a button to clear all values from localStorage ('Forget my answers'... something like that).